DP1.10058.001/Standard.calculations.txt

```diff
@@ -1,6 +1,6 @@
-Species-specific cover observed in 1m^2 subplots is directly available in the table div_1m2Data. Subplots at which observations were made are identified by the subplotID field (e.g., 31.1.1, 40.3.1), and are associated with specific plots (identified d by the plotID field, e.g., HARV_001) within the site (Figure 1).  
+Species-specific cover observed in 1m^2 subplots is directly available in the table div_1m2Data. Subplots at which observations were made are identified by the subplotID field (e.g., 31_1_1, 40_1_3), and are associated with specific plots (identified by the plotID field, e.g., HARV_001) within the site (Figure 1).  
```

Comment:
Add a link to neonPlants to the Standard calculations section as well?

Reply:
done

Reply:
@cklunch edits made, thanks for the good suggestions!